sqlite-android/build.gradle

```
@@ -8,13 +8,13 @@ apply plugin: 'com.jfrog.bintray'
 apply plugin: 'de.undercouch.download'
 
 group = 'io.requery'
-version = '3.34.1'
+version = '3.35.1.1'
```

This is bogus for the purposes of the main repo. This repo is still on 3.35.1 so should be more like this:

```suggestion
version = '3.35.1'
```

...and then the commit that puts this PR in place should be tagged `3.35.1`, with the tag pushed to github, in order for jitpack to build it